examples/ColumnGroupsExample.js

```diff
@@ -29,41 +29,56 @@ class ColumnGroupsExample extends React.Component {
         rowsCount={dataList.getSize()}
         width={1000}
         height={500}
+        columnsCount={4}
+        getColumn={(i) =>
```

Comment:
This is gonna create the list of columns on every render.
This is fine for now, but we should make a follow up PR later to either memoize this or simply move out the initialization to the constructor.